poker-pals/src/components/ChipDistributionCalculator.vue

```diff
@@ -0,0 +1,164 @@
+<template>
+    <div>
+        <div class="box">
+            <input class="rank" type="text" id="player1r" size="1">
+            <input class="rank" type="text" id="player2r" size="1">
+            <input class="rank" type="text" id="player3r" size="1">
+            <input class="rank" type="text" id="player4r" size="1">
+            <input class="rank" type="text" id="player5r" size="1">
+            <input class="rank" type="text" id="player6r" size="1">
+        </div>
+        <div class="box">
+            <input type="text" id="player1" size="10">
+            <input type="text" id="player2" size="10">
+            <input type="text" id="player3" size="10">
+            <input type="text" id="player4" size="10">
+            <input type="text" id="player5" size="10">
+            <input type="text" id="player6" size="10">
+        </div>
+        <div class="box">
+            <button v-on:click="calculate()">get number</button>
+            <p id="result">0</p>
+        </div>
+    </div>
+</template>
+
+<style scoped>
+
+    .box{
+        width: 923px;
+        margin: auto;
+    }
+    .rank{
+        margin: 0px 0px 0px 53px;
+    }
+
+</style>
+
+<script>
+
+export default {
+    name: "ChipDistributionCalculator",
+    mounted(){
+
+    },
+    methods:{
+        calculate(){
+            
+            let players = [
+                {id: 1, rank: 0x500000, moneyPot: 10, returnPot: 0},
+                {id: 2, rank: 0x900000, moneyPot: 3, returnPot: 0},
+                {id: 3, rank: 0x600000, moneyPot: 3, returnPot: 0},
+                {id: 4, rank: 0x900000, moneyPot: 2, returnPot: 0},
+                {id: 5, rank: 0x700000, moneyPot: 1, returnPot: 0},
+                {id: 6, rank: 0x800000, moneyPot: 4, returnPot: 0}
+            ];
+            
+            let slots = [];
+            slots.push([]);
+            slots.push([]);
+            slots.push([]);
+            slots.push([]);
+            slots.push([]);
+            slots.push([]);
+            /*
+            let players = [];
+            for(let i = 0; i < 6; i++){
+                let rank = document.getElementById('player' + (i + 1).toString() + 'r').value;
+                let money = document.getElementById('player' + (i + 1).toString()).value;
+                players.push({id: i+1, rank: rank, moneyPot: money, returnPot: 0});
+            }
+            */
```

Comment:
Does the commented out code provide any value? If not, might be good to remove

Reply:
originally i was testing it by manually typing in values on the web page but switched over to hard coded values. ill remove it